routes/tweetRoutes.js

```diff
@@ -11,9 +11,9 @@ router.get('/' , getTweets);
 
 router.post('/tweets', ensureAuthorizated, createTweet);
 
-router.patch('/update', ensureAuthorizated,  updateTweets);
+router.patch('/update/:id', ensureAuthorizated,  updateTweets);
```

Comment:
Change `/update/:id` to `/:id`. The PATCH method is already indicating an update

Reply:
ok